get.sh

```diff
@@ -322,6 +349,9 @@ parseCommandLineArgs "$@"
 if [[ "$SDKDIR" != "" ]]; then
 	getBinaryOpenjdk
 fi
+if [[ "$TESTDIR" != "" ]]; then
+	getOpenJDKSources
+fi
```

Comment:
$TESTDIR should never be empty. I would suggest moving the condition `if [ "$SDK_RESOURCE" == "customized" ] && [ "$CUSTOMIZED_SDK_SOURCE_URL" != "" ]` out of function `getOpenJDKSources()` as

```
if [ "$SDK_RESOURCE" == "customized" ] && [ "$CUSTOMIZED_SDK_SOURCE_URL" != "" ]; then
         getOpenJDKSources
fi

Reply:
Thanks, fixed.